content/docs/_index.md

```diff
@@ -8,6 +8,9 @@ menu:
     weight: 20
 no_list: true
 ---
+{{% pageinfo color="primary" %}}
+The CSM 1.7.1 is applicable to helm based installations of PowerFlex driver.
```

Comment:
Should it be CSI PowerFlex driver here?

Reply:
No @rensyct the requester asked to mention only PowerFlex and avoid csi